MultiTable/src/SubjectGroups.js

```diff
@@ -14,13 +14,18 @@ const d3c = require('./lib/d3-collection.v1.min.js');
 const SubjectGroup = require('./SubjectGroup');
 const Utils = require('./Utils');
 
+/*
+ * Return the absolutePath one level up from the absolutePath or sample name
+ * specified. Just the first part of the condition is required, other probably not
```

Comment:
what do you mean by this?

Reply:
`Just the first part of the condition is required, other probably not` 
I just wrote this for myself inititally when I saw line 22 but later realized that I was wrong. I will remove this comment.